pages/common/quarto.md

```diff
@@ -0,0 +1,36 @@
+# quarto
+
+> An open-source scientific and technical publishing system built on Pandoc.
+> More information: <https://quarto.org/>.
+
+- Create a Quarto project:
+
+`quarto create-project {{path/to/project}}`
+
+- Create a new website project:
+
+`quarto create-project **{{destination_directory}}** --type {{website}}`
+
+- Create a new book project:
+
+`quarto create-project **{{destination_directory}}** --type {{book}}`
+
+- Render an R Markdown file to HTML:
+
+`quarto render {{path/to/file.rmd}} --to html`
+
+- Render a Quarto file to HTML:
+
+`quarto render {{path/to/file.qmd}} --to html`
+
+- Render a Jupyter notebook to HTML:
+
+`quarto render {{path/to/file.ipynb}} --to html`
+
+- Render a document to PDF:
+
+`quarto render {{path/to/file.qmd}} --to pdf`
+
+- Render a document to Microsoft docx document:
+
+`quarto render {{path/to/file.qmd}} --to docx`
```

Comment:
```suggestion
- Create a Quarto project:

`quarto create-project {{path/to/project}}`

- Create a new website project:

`quarto create-project {{path/to/project}} --type {{website}}`

- Create a new book project:

`quarto create-project {{path/to/project}} --type {{book}}`

- Render an R Markdown file to HTML:

`quarto render {{path/to/file.rmd}} --to {{html}}`

- Render a Quarto file to HTML:

`quarto render {{path/to/file.qmd}} --to {{html}}`

- Render a Jupyter notebook to HTML:

`quarto render {{path/to/file.ipynb}} --to {{html}}`

- Render a document to PDF:

`quarto render {{path/to/file.qmd}} --to {{pdf}}`

- Render a document to Microsoft docx document:

`quarto render {{path/to/file.qmd}} --to {{docx}}`
```

Reply:
I think I managed to fix it. Do not hesitate if I missed something.

Reply:
Well, you didn't apply this suggestion. The argument after `--too` as to be in `{{ }}` and it should be `{{path/to/project}}` instead of `**{{destination_directory}}**`.